code/game/gamemodes/wizard/artefact.dm

```diff
@@ -199,5 +211,5 @@
 
 /obj/item/weapon/scrying/attack_self(mob/user as mob)
 	user << "\blue You can see...everything!"
-	visible_message("\red <B>[usr] stares into [src], their eyes glazing over.</B>")
+	visible_message("\red <B>[user] stares into [src], their eyes glazing over.</B>")
```

Comment:
yeah i was gonna fix all of them in the file but i saw so many that i think my head exploded...i just changed that one user..then looked all the preexisting stuff on the file....